flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/endpoint/hive/HiveServer2EndpointITCase.java

```diff
@@ -137,7 +137,7 @@ public class HiveServer2EndpointITCase extends TestLogger {
             new HiveServer2EndpointExtension(SQL_GATEWAY_SERVICE_EXTENSION::getService);
 
     @BeforeAll
-    public static void setup() throws Exception {
+    static void setup() throws Exception {
```

Comment:
This class shouldn't extends `TestLogger`, and the public can be removed at class level.

Note: the public can be removed for all test methods.

TBH, these changes should be finished in the previous PR, but they are forgot. Thank you for the updating!